ui/src/onboarding/components/SigninForm.scss

```diff
@@ -1,8 +1,38 @@
-.cf-panel.signin-panel {
+.signin-page {
+  .cf-funnel-page--content {
+    display: flex;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .cf-dapper-scrollbars--content {
+    display: flex !important;
+    align-items: center;
+  }
+}
+
+.cf-panel.signin-page--panel {
   width: 300px;
 
   .cf-panel--body {
     padding: $ix-marg-d;
     text-align: center;
   }
-}
\ No newline at end of file
+}
+
+.signin-page--logo {
+  width: 70%;
+  height: auto;
+  margin-top: 1em;
+  margin-bottom: 3em;
+}
+
+.signin-page--cubo {
+  background-image: url('../../assets/images/auth-logo.svg');
+  background-size: 100% 100%;
+  background-position: center center;
+  background-repeat: no-repeat;
+  width: 100px;
+  height: 100px;
+  display: inline-block;
+}
```

Comment:
ninor nit: wanna alphabetize these selectors?

Reply:
I can do that, though makes me think we could benefit from more thorough CSS formatting. I think prettier supports that? Maybe a nice little tooling improvement

Reply:
@alexpaxton deffo. I usually just say "alphabetize absent any other organizing scheme. would be rad to have one in place"